projects/packages/stats-admin/tests/php/Admin_Post_List_Test.php

```diff
@@ -51,6 +51,36 @@ public function test_add_stats_post_table_adds_stats_column() {
 		$this->assertEquals( 'Stats', $columns_with_stats['stats'] );
 	}
 
+	/**
+	 * Test if the stats column is added when comments is the first column of the table.
+	 *
+	 * @return void
+	 */
+	public function test_adds_stats_column_when_comments_is_the_first_column() {
```

Comment:
I could write a data provider to avoid having two identical tests with different data sets. But... I'm not sure if it's worth the effort.